help.go

```diff
@@ -10,8 +10,10 @@ import (
 )
 
 const (
-	defaultHelpDescription = "print this help menu"
-	defaultHelpMessage     = "Usage of __EXEC__:"
+	defaultHelpDescription    = "print this help menu"
+	defaultHelpMessage        = "Usage of __EXEC__:"
+	defaultVersionDescription = "print the program version"
+	defaultVersionMessage     = "Version: %s\n"
```

Comment:
I personally prefer to just print the version without a prefix. At least by default not force prefix anything. A `--version` printing just the version is a lot more useful for Bash scripts doing assertions on it.

Reply:
Good point!